### ISSUE TITLE
Improve Visibility of "Archive" and "Star" Icons

### ISSUE THREAD

**Issue by skygreenslade** [red-coracle/episodes#17]
The "Archive" and "Star" icons are not visible against the dark banner of some shows.

**Comment by robinpaulson** [comment]
Yes, I struggle to locate them too. Perhaps a contrasting border around each?

**Comment by OliverLew** [comment]
Maybe a better icon is required.

Most icons are white with bold lines, but the archive icon is gray and narrow:

![IMG_20211122_104901](https://user-images.githubusercontent.com/12032219/142793902-9684a2e3-6502-49ea-84de-61ef7e7eeeb5.jpg)

Version 0.15.0
